src/clicache/src/CacheFactory.cpp

```diff
@@ -84,6 +85,12 @@ namespace Apache
 					//TODO::split
           SafeConvertClassGeneric::SetAppDomainEnabled(appDomainEnable);
 
+          if (appDomainEnable)
+          {
+            // Register managed AppDomain context with unmanaged.
+            apache::geode::client::createAppDomainContext = &Apache::Geode::Client::createAppDomainContext;
```

Comment:
Does moving the registration of the managed app domain context to cache factory from distributed system make it available sooner?